JSDemos/Demos/Charts/AxisLabelsOverlapping/description.md

```diff
@@ -1 +1,16 @@
-In&nbsp;this demo, argument axis labels overlap due to&nbsp;their length. You can decide how to&nbsp;resolve the overlapping by&nbsp;choosing one of&nbsp;the overlapping behavior modes.
\ No newline at end of file
+In this demo, argument axis labels overlap due to their length. To resolve overlapping, you can specify one of the [label](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/label/).[overlappingBehavior](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/label/#overlappingBehavior) modes:
+
+- `none`    
```

Comment:
This item does not suit either 'resolving' or 'avoiding' overlapping labels since it leaves overlapping lables